src/containers/my-quizzes/create-or-edit-quiz-container/CreateOrEditQuizContainer.tsx

```diff
@@ -236,14 +249,19 @@ const CreateOrEditQuizContainer = ({
           />
         )}
         <Box sx={styles.functionalButtons}>
-          <AppButton
-            onClick={onOpenCreateQuestion}
-            size={SizeEnum.ExtraLarge}
-            variant={ButtonVariantEnum.Tonal}
-          >
-            {t('myResourcesPage.quizzes.createNewQuestion')}
-            <EditIcon fontSize={SizeEnum.Small} />
-          </AppButton>
+          {isCreationOpen ? (
+            createNewQuestionButtonInactive
+          ) : (
+            <AppButton
+              onClick={onOpenCreateQuestion}
+              size={SizeEnum.ExtraLarge}
+              variant={ButtonVariantEnum.Tonal}
+            >
+              {t('myResourcesPage.quizzes.createNewQuestion')}
+              <EditIcon fontSize={SizeEnum.Small} />
+            </AppButton>
+          )}
```

Comment:
I suggest moving code part with ternary operator to separate variable, for example called `CreateQuestionButton` to avoid polluting our jsx

Reply:
Moved in a separate variable.